### PR TITLE
Minor changes to make bot-calling-meeting work

### PR DESCRIPTION
**Pull request by eoinobrien**
Minor enough changes I made to be able to test this app. I've intentionally made few changes in this PR so that I can get this completed quickly.

I will return with more changes in the future to make this app and the media bot docs easier to understand.

### CODE REVIEW

File: samples/bot-calling-meeting/csharp/README.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ Calling and Meeting Bot provides basic functionality like Create Call, Join a ca
   - Navigate to `samples/bot-calling-meeting/csharp` folder
   - Select `CallingBotSample.csproj` file
 
-1) Run ngrok - point to port 3978
+1) Run ngrok - point to port 44346
```
Comment (aosolis) [inline samples/bot-calling-meeting/csharp/README.md:45]: why the change to a different port?

Comment (eoinobrien) [reply samples/bot-calling-meeting/csharp/README.md:45]: This is the default port vs gives me when I launch. I do plan to add a launchsettings to actually make this consistent across deployments

**Review by aosolis — APPROVED** [review]

File: samples/bot-calling-meeting/csharp/README.md (hunk 1 above)
Comment (Prasad-MSFT) [reply samples/bot-calling-meeting/csharp/README.md:45]: If you run the sample in IIS Express it will run in port 44346.
image

However it will run in port 3978 if CallingBotSample is selected.
image

Comment (Wajeed-msft) [reply samples/bot-calling-meeting/csharp/README.md:45]: @Prasad-MSFT - I think on of the step by step guide refers to 3978 port number. Could you please check? 
https://learn.microsoft.com/en-us/microsoftteams/platform/sbs-calling-and-meeting?tutorial-step=2
image

Comment (eoinobrien) [reply samples/bot-calling-meeting/csharp/README.md]: I did revert this change to what 3978 for now

**Review by Wajeed-msft — APPROVED** [review]
